ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -206,6 +206,18 @@
     reason="Test runs ONLY on OSD or ROSA cluster",
 )
 
+ms_provider_required = pytest.mark.skipif(
+    (config.ENV_DATA["platform"].lower() not in MANAGED_SERVICE_PLATFORMS)
```

Comment:
Is it possible that platform would NOT be managed service AND cluster type would be provider? Don't you want to skip the test if platform is not managed service OR cluster type != provider? Sorry If I misunderstand the logic.

Reply:
Good catch. Thanks, @ebondare . Modified the conditons